Command/CreateDatabaseDoctrineCommand.php

```diff
@@ -64,6 +65,10 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         /** @psalm-suppress InvalidArrayOffset Need to be compatible with DBAL < 4, which still has `$params['url']` */
         unset($params['dbname'], $params['path'], $params['url']);
 
+        if ($connection->getDatabasePlatform() instanceof PostgreSQLPlatform) {
+            $params['dbname'] = 'postgres';
+        }
```

Comment:
Why do we unset `dbname` on line 66, only to set it again here? Also, I don't understand the comment on line 64, should it read "as we have already extracted" instead of "and we have already extracted"?

Reply:
For the first point, it's because I assume unsetting dbname might still be needed for other db platforms. For the second point: I guess so.